app/src/main/java/co/nimblehq/alpine/sample/CameraCaptureActivity.kt

```diff
@@ -48,6 +76,8 @@ class CameraCaptureActivity : ComponentActivity() {
             }
 
             pvCameraCaptureViewFinder.post {
+                observeCapture()
+                updateCameraUi()
                 configureCamera()
```

Comment:
Any reason why these calls are inside the `post` block? Not sure I understand why, seems to work fine on without it 😅